apps/meteor/app/livechat/server/lib/Livechat.js

```diff
@@ -560,10 +564,15 @@ export const Livechat = {
 		return false;
 	},
 
+	async afterRemoveAgent(user) {
+		await callbacks.run('livechat.afterAgentRemoved', { agent: user });
```

Comment:
Any specific reason to put this in a callback? Since it's being called only from here, it'd be clear to just put that code here instead of creating one callback more.